org.kicad_pcb.KiCad.json

```diff
@@ -39,36 +40,74 @@
                 }
             ]
         },
+        {
+            "name": "OCE",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/tpaviot/oce/archive/OCE-0.18.3.tar.gz",
+                    "sha256": "c553d6a7bf52f790abc3b6bb7a1e91a65947e92a426bb1a88a11960c31f0966c"
+                }
+            ],
+            "buildsystem": "cmake",
+            "config-opts": [
+                "-DOCE_INSTALL_PREFIX=/app"
+            ]
+        },
+        {
+            "name": "glm",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/g-truc/glm/releases/download/0.9.9.0/glm-0.9.9.0.zip",
+                    "sha256": "e1c707407c43589e8eeb8b69b902f1a34aaaa59bda1ca144181c2d2d6e531246"
+                }
+            ],
+            "buildsystem": "cmake"
+        },
+
+
         {
             "name": "boost",
             "sources": [
                 {
                     "type": "archive",
-                    "url": "http://sourceforge.net/projects/boost/files/boost/1.64.0/boost_1_64_0.tar.bz2",
-                    "sha256": "7bcc5caace97baa948931d712ea5f37038dbb1c5d89b43ad4def4ed7cb683332"
+                    "url": "http://sourceforge.net/projects/boost/files/boost/1.66.0/boost_1_66_0.tar.bz2",
+                    "sha256": "5721818253e6a0989583192f96782c4a98eb6204965316df9f5ad75819225ca9"
                 }
             ],
             "buildsystem": "simple",
             "build-commands": [
                 "./bootstrap.sh --prefix=/app",
-                "./b2 install --with-context --with-date_time --with-filesystem --with-iostreams --with-locale --with-program_options --with-regex --with-system --with-thread --prefix=/app"
+                "./b2 install --with-context --with-date_time --with-filesystem --with-iostreams --with-locale --with-program_options --with-regex --with-system --with-thread --with-test --prefix=/app"
             ]
         },
         {
-            "name": "kicad",
+            "name": "ngspice",
+            "rm-configure": true,
+            "config-opts": [ "--enable-shared", "--disable-static", "--without-x", "--disable-silent-rules" ],
```

Comment:
looks like this also needs "--with-ngshared" according to https://flathub.org/builds/#/builders/26/builds/343

Reply:
The `get_libngspice_so.sh` script also mentioned in the log uses the following command to configure ngspice (see https://github.com/KiCad/kicad-source-mirror/blob/master/scripts/get_libngspice_so.sh):

`./configure --with-ngshared --enable-xspice --enable-cider ${CFG_OPTIONS}`

where `${CFG_OPTIONS}` expands to `--enable-openmp` on linux platforms.